dags/datasets/toronto-s-dashboard-key-indicators.py

```diff
@@ -416,7 +416,7 @@ def send_new_records_notification(**kwargs):
         download_file_task_id="get_measure",
         resource_task_id="get_or_create_resource",
         upload_to_ckan=False,
-        sync_timestamp=False,
+        sync_timestamp=True,
```

Comment:
We can ignore this file entirely - this change was already pushed from a separate branch previously